### PR TITLE
create clock component for reflect with reducers

### PR DESCRIPTION
**Pull request by Alexander-Wen**
will worry about styling and making it look good later

### CODE REVIEW

File: src/components/Clock.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  componentDidMount () {
+    this.interval = setInterval(() => {
+      const d = new Date();
+      const time = {
+        hour: d.getHours(),
+        minute: d.getMinutes(),
+      };
+      ClockActions.updateTime(time);
+      this.forceUpdate();
+    }, 60000)
```
Comment (Alexander-Wen) [inline src/components/Clock.js:11]: update state every minute

File: src/components/Clock.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  render() {
+    const dateTime = JSON.parse(ClockActions.getState().toString().replace(/Map/g,''));
+    const hour = dateTime.time.hour % 12;
+    const minute = dateTime.time.minute;
+    const day = dateTime.date.day;
+    const weekday = dateTime.date.weekday;
+    const month = dateTime.date.month;
+
+    return (
```
Comment (Alexander-Wen) [inline src/components/Clock.js]: will worry about making things pretty later

**Comment by AdrianMachado** [comment]
:shipit:

**Comment by AdrianMachado** [comment]
:shipit:

**Pull request merged**